src/internal-links/handler.js

```diff
@@ -106,7 +108,67 @@ export async function internalLinksAuditRunner(auditUrl, context, site) {
   };
 }
 
+async function convertToOpportunity(auditUrl, auditData, context) {
+  const {
+    dataAccess,
+    log,
+  } = context;
+
+  const opportunities = await dataAccess.Opportunity.allBySiteIdAndStatus(auditData.siteId, 'NEW');
+  let opportunity = opportunities.find((oppty) => oppty.getType() === AUDIT_TYPE);
+
+  if (!opportunity) {
+    const opportunityData = {
+      siteId: auditData.siteId,
+      auditId: auditData.id,
+      runbook: 'https://adobe.sharepoint.com/sites/aemsites-engineering/Shared%20Documents/3%20-%20Experience%20Success/SpaceCat/Runbooks/Experience_Success_Studio_Broken_Internal_Links_Runbook.docx?web=1',
+      type: AUDIT_TYPE,
+      origin: 'AUTOMATION',
+      title: 'Broken Internal Links',
```

Comment:
Can you update it to: "Broken internal links found", same as mentioned in the mocks

Reply:
done